pilot/pkg/model/authorization.go

```diff
@@ -114,6 +116,18 @@ func (policy *AuthorizationPolicies) IsGlobalPermissiveEnabled() bool {
 		policy.rbacConfig.EnforcementMode == rbacproto.EnforcementMode_PERMISSIVE
 }
 
+// ListNamespacesOfToV1alpha1Policies returns all namespaces that have V1alpha1 policies.
+func (policy *AuthorizationPolicies) ListNamespacesOfToV1alpha1Policies() []string {
```

Comment:
This doesn't seem appropriate in `model`. Is there no other existing method that exposes enough info to calculate the namespaces? Maybe just get all Policies and extract the unique namespaces for example? I would rather have istioctl do more work (in a command that performance is really not relevant at all?) than have an extra method exposed here. If its not feasible though then its fine

Reply:
I think it can do done the other way, but why is it bad to include this method in the model?

Reply:
I think model should be pretty generic, `ListNamespacesOfToV1alpha1Policies` is a hyper specific function just for istioctl.

Reply:
istioctl happens to use this code, but rbac filter conversion can also use this as well. List all namespaces is a pretty generic (just like the existing list all ServiceRole method), I would say.